src/Selenium2Driver.php

```diff
@@ -762,9 +752,13 @@ public function click($xpath)
         $this->clickOnElement($this->findElement($xpath));
     }
 
+    /**
+     * @param Element $element
+     */
     private function clickOnElement(Element $element)
     {
-        $this->wdSession->moveto(array('element' => $element->getID()));
+        //$element->click(); // TODO why was this needed?
```

Comment:
What this comment is for? The `click` is called because method is called `clickOnElement` 😄